doc/connectivity/networking/api/lwm2m.rst

```diff
@@ -516,6 +520,11 @@ The events are prefixed with ``LWM2M_RD_CLIENT_EVENT_``.
        If sending a message fails, it will be retried.
        If the retry counter reaches its limits, this event will be triggered.
      - No actions needed, client will do a re-registrate automatically.
+   * - 12
```

Comment:
This value doesn't seem to correspond to the actual enum value.

Reply:
Ah I need then change.